render-ws-java-client/src/test/java/org/janelia/render/client/parameter/StackIdWithZParametersTest.java

```diff
@@ -66,6 +66,12 @@ public void testGetStackIdList() throws IOException {
                 .filter(stackId -> stackId.getProject().equals(projectB)).count();
         Assert.assertEquals("all stacks should be from project B (naming group should override default)",
                             6, projectBStackCount);
+
```

Comment:
`testGetStackIdList()` should probably also be split in multiple tests, but this is not related to the PR under review.

Reply:
Yeah - I'm ok with it either way.  If you feel compelled to split it up, that's fine - I'll leave it to you to do :).